docs/DeveloperGuide.md

```diff
@@ -1,34 +1,503 @@
 # Developer Guide
 
-## Design & implementation
+![Trippie Logo](https://i.imgur.com/jSwGL7O.png)
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+## Table of Content
```

Comment:
The hyperlinks for each section in the table of contents are not working on github pages. Or is there not suppose to be a hyperlink to the sections?